analyze/analyze.rs

```diff
@@ -358,8 +363,25 @@ pub fn dominators(
     items.compute_dominator_tree();
     items.compute_retained_sizes();
 
+    /// Get the Id of the item with the given name.
+    fn get_item_id(items: &mut ir::Items, name: &str) -> Option<ir::Id> {
```

Comment:
This seems like a generally useful utility method to add to `ir::Items`:

```rust
impl Items {
    ...

    fn get_item_by_name(&self) -> Option<&Item> {
        ...
    }
}
```

Then, if we just need the id, we can map over the returned option like `items.get_item_by_name("blah").map(|item| item.id())`.